package-res/resources/web/pentaho/lang/EventSource.js

```diff
@@ -325,10 +326,11 @@ define([
         result = this._emit(event);
```

Comment:
On the whole `_emitSafe` approach, try/catch should be by listener. But I already saw @carlosrusso doing something in that path.